.env.sample

```diff
@@ -3,6 +3,10 @@ DB_USER=
 DB_PASS=
 DB_HOST=
 
```

Comment:
Could you add a comment explaining what these variables are for? Also, I think I'd like to stick with `SUPERUSER` instead of `ADMIN`.

Reply:
Also, `USERNAME` is not correct - it's their full name. Maybe just change to `SUPERUSER_NAME`.

Reply:
Will do